server/src/controllers/v1/cards.js

```diff
@@ -1,12 +1,18 @@
 const { Card } = require('../../models')
 
 module.exports = {
-  create(req, res) {
-    return Card
+  create: (req, res) => {
+    Card
       .create({
         description: req.body.description,
+        raw: true,
```

Comment:
@DDKatch why is it needed?

Reply:
@AleksSenkou in order to get array, in other way you should use `cards.dataSources` instead of just cards.